src/app/clusters/basic/basic.cpp

```diff
@@ -318,6 +318,101 @@ void emberAfBasicClusterServerInitCallback(chip::EndpointId endpoint)
     }
 }
 
+EmberAfStatus BasicClusterServerGetVendorIdAttribute(chip::EndpointId endpointId, chip::VendorId & value)
+{
+    return Attributes::VendorID::Get(endpointId, &value);
+}
+
+EmberAfStatus BasicClusterServerSetVendorIdAttribute(chip::EndpointId endpointId, chip::VendorId value)
+{
+    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
+
+    chip::VendorId currentValue;
+    status = Attributes::VendorID::Get(endpointId, &currentValue);
+    VerifyOrDie(EMBER_ZCL_STATUS_SUCCESS == status);
```

Comment:
Why VerifyOrDie if the function returns a status... The status is never left to be returned. Please remove VerifyOrDie everywhere here... It should be the caller application's choice what to do about that here.